### DISCUSSION TITLE
CP30没开票的话是看不了选票信息吗

### DISCUSSION THREAD

**Discussion by sacrificerrr** [mikumifa/cppTickerBuy#25] [提问专区]
![G2$}LJ{K_8B`VA5I23NYFYI](https://github.com/user-attachments/assets/550a1d65-3472-4207-b04a-970a4e59a1d7)
如图右下角那个选票一直点不了，微软、谷歌、火狐浏览器都用过了

**Comment by mikumifa** [comment]
是的，开票之后应该才会有
